README.md

```diff
@@ -271,6 +321,63 @@ terraform {
      }
  }
 ```
+## Helpful Modules
+### Datastore ("datastore")
+
+A datastore module is included to quick spin up an AWS RDS instance and provide the connection string to any resource that needs it.
+
+Example usage:
+
+```hcl
+module "datastore" {
+    source = "./terraform_modules/datastore"
+
+    providers = {
+        aws = aws.us-west-2
+    }
+    
+    name = "cloud_inquisitor_datastore"
+    environment = "dev"
+    region = "us-west-2"
+    version_str = "v0_0_0"
+    skip_final_snapshot = true
+
+    datastore_vpc = "vpc-id"
+    datastore_connection_cidr = [
+        "0.0.0.0/0"
+    ]
+
+    datastore_subnets = [
+        "subnet-1",
+        "subnet-2",
+        "subnet-3"
+    ]
+    datastore_tags = {
+        "Name": "cloud-inqusitor-datastore",
+        "owner": "your@email.com",
+        "accounting": "$(whoami)"
+    }
+}
+```
+
+## Project Role ("project_role")
+
+The Project Role module creates an IAM role with all of the necessary permissions to access resources used by CloudInquisitor. Although there are lots of default permissions; the list only include the actions taken via the CloudInquisitor code and could be futher scoped/customized if only parts of the stack are used.
+
+This module also creates a secondary role with identical permissions but also includes a trust relation that allows for the primary role to assume role into the secondary. If you are a multi-account organization, this role can be added to all accounts to allow Cloud Inquisitor access.
+
+Example usage:
+```hcl
+module "project_role" {
+    source = "./terraform_modules/project_role"
+
+    name = "cinq-next"
+    environment = "dev"
+}
+```
+
+### Vault
```

Comment:
do we planned to add vault intro later?

Reply:
added the necessary comments/description of how to use vault